### PR TITLE
Adds Governor 1.0.4 announcement.

### PR DESCRIPTION
**Pull request by dipak-pawar**
This change is [Reviewable](https://reviewable.io/reviews/arquillian/arquillian.github.com/348)

### CODE REVIEW

File: blog/arquillian-governor-1.0.4.Final.textile [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+We are proud to announce new features in this release for Arquillian Governor:
+
+h3. New Features
+
+- <i class="icon-star"></i> Arquillian Recorder integration with Arquillian Governor:=
+
+p. In this release we are providing recorder integration with Jira, GitHub & Redmine Governor.
+
+p. Arquillian Recorder project brings neat reports of your Arquillian tests. Now we are providing an integration to the project so that generated report contains information about JIRA, GitHub and Redmine properties such as Issue URL, Detector & it's strategy used for environment, force value.
```
Comment (bartoszmajsak) [inline blog/arquillian-governor-1.0.4.Final.textile]: @dipak-pawar @lordofthejars this paragraph is a bit blurry from the end-user perspective I believe. Is it about recorder or governor? What are these values used for?

> properties such as Issue URL, Detector & it's strategy used for environment, force value.

Can we rephrase it and explain what does it really stand for and how is it used?

**Comment by bartoszmajsak** [comment]
Thanks @dipak-pawar. I have just one comment about the first paragraph.

File: blog/arquillian-governor-1.0.4.Final.textile (hunk 1 above)
Comment (dipak-pawar) [inline blog/arquillian-governor-1.0.4.Final.textile]: @Bartosz Updated announcement

**Comment by bartoszmajsak** [comment]
Thanks, it's already live! @d624afea067418fd60f37db2b6911d2f68674e31

**Pull request closed without merging**